plugin/src/main/java/io/spine/tools/gradle/bootstrap/JavaExtension.java

```diff
@@ -60,6 +60,10 @@ private JavaExtension(Builder builder) {
     @Override
     void enableGeneration() {
         super.enableGeneration();
+
+        dependOn(SpineDependency.testlib(), testImplementation);
+        dependOn(SpineDependency.testUtilTime(), testImplementation);
+
```

Comment:
Please remove the empty lines. If you'd like to make this logic stand out, please extract it into a method.